Library/Formula/ipe.rb

```diff
@@ -38,7 +38,7 @@ class Ipe < Formula
 
   def install
     cd 'src' do
-      system "make", "IPEPREFIX=#{prefix}"
+      system "make", "IPEPREFIX=#{HOMEBREW_PREFIX}"
```

Comment:
No worries. Bumping the version allowed me to fix a few of the other bugs that were lying around the formula, which was good. Upstream have said they will never officially support ` ipe ` for OS X, which is deeply frustrating, but as long as it works, we're happy to keep carrying it.